### PR TITLE
Fix #10478: tags updation issue for schemaFields and tasks

### LINKED ISSUE #10478: UI: tags adding and removing flow for schemaFields and tasks not working properly
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
UI

**Describe the bug**

- [ ] For schema fields of `Topic` entities, no API call is being made while trying to remove tags

https://user-images.githubusercontent.com/51777795/223743137-2bca52cc-eae7-43c6-b182-80c07c51d963.mov

- [ ] For tasks of `Pipeline` entities, no API calls are being made while trying to add tags

https://user-images.githubusercontent.com/51777795/223744225-f53c6685-646c-46af-bb8e-0bf6eec4e7ef.mov

**To Reproduce**

Screenshots or steps to reproduce

**Expected behavior**

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on fixing #10478 
Added unit tests and cypress tests for the same

For issues with adding tags to the tasks for pipeline entities, the API response for the PATCH API is received as successful but the tags are not actually being added to the tasks. I have raised an issue for the same here -> #10482 

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">

https://user-images.githubusercontent.com/51777795/224067052-e2e74214-a94a-4189-95c4-7f116480c4a8.mov

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx [hunk 1]
```diff
@@ -569,17 +584,18 @@ const PipelineDetails = ({
...
-              icon="external-link"
-              width="16px"
-            />
-          </Link>
-        ),
+          ) : (
+            <Link
+              className="flex items-center gap-2"
+              target="_blank"
+              to={{ pathname: pipelineHostPort + record.taskUrl }}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:594]: I think we decide to not to have any logic for link creation?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/TagsAddRemove.spec.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    cy.login();
+  });
+
+  TAGS_ADD_REMOVE_ENTITIES.map((entityDetails) =>
+    it(`Adding tag to ${entityDetails.entity} entity should work properly`, () => {
+      visitEntityDetailsPage(
+        entityDetails.term,
+        entityDetails.serviceName,
+        entityDetails.entity
+      );
```
[the remark this answers is not in the document]
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/TagsAddRemove.spec.js:49]: We are adding and removing tags for `table` in some tests but we were not checking that for every entity. As we stumbled upon this issue for particular entities which were not caught in the existing cypress tests, I thought of adding the tests for other entities as well

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:594]: Right. Thanks for catching this. I missed changing this later discussion. Will do the changes.

**Review by chirag-madlani — DISMISSED** [review]
Thanks

**Comment by harshach** [comment]
@aniketkatkar97 initial testing looks good to me. I need more time to test it with lot more data and filters. But lets merge it in for now and we can incrementally address it cc @chirag-madlani

**Review by harshach — APPROVED** [review]

**Pull request merged**